commercetools-models/src/main/java/io/sphere/sdk/carts/LineItemDraftBuilder.java

```diff
@@ -0,0 +1,29 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.ByIdVariantIdentifier;
+import io.sphere.sdk.taxcategories.ExternalTaxRateDraft;
+import io.sphere.sdk.types.CustomFieldsDraft;
+
+import javax.annotation.Nullable;
+import javax.money.MonetaryAmount;
+
+public final class LineItemDraftBuilder extends LineItemDraftBuilderBase<LineItemDraftBuilder> {
+
+    LineItemDraftBuilder(@Nullable CustomFieldsDraft custom, @Nullable Reference<Channel> distributionChannel, @Nullable MonetaryAmount externalPrice, @Nullable ExternalTaxRateDraft externalTaxRate, @Nullable ExternalLineItemTotalPrice externalTotalPrice, String productId, @Nullable Long quantity, @Nullable String sku, @Nullable Reference<Channel> supplyChannel, @Nullable Integer variantId) {
+        super(custom, distributionChannel, externalPrice, externalTaxRate, externalTotalPrice, productId, quantity, sku, supplyChannel, variantId);
+    }
+
+    /**
+     * Creates a new object initialized with the given values.
+     *
+     * @param variantIdentifier initial value for the {@link LineItemDraft#getProductId()} and {@link LineItemDraft#getVariantId()} properties
+     * @param quantity          initial value for the {@link LineItemDraft#getQuantity()} property
+     * @return new object initialized with the given values
+     */
+    public static LineItemDraftBuilder ofVariantIdentifier(final ByIdVariantIdentifier variantIdentifier, @Nullable final Long quantity) {
```

Comment:
Here we should add another factory method that accepts a `BySkuVariantIdentifier` as input.